src/index.ts

```diff
@@ -0,0 +1,77 @@
+import Eth from 'ethjs';
+import registryMap from './registry-map.json';
+import abi from './abi.json';
+
+interface HttpProvider {
+  host: string;
```

Comment:
Comment for posterity: We need provider types in DefinitelyTyped one day...

cc: @brad-decker